### PR TITLE
Add new sh-mikem bio

### LINKED ISSUE #56: Add bio for sh-mikem
#### Name:
Mike

#### Location:
Sunnyvale, CA

#### Favorite food:
Pizza

#### A Hobby:
Watch TV

### PR DESCRIPTION
**Pull request by sh-mikem**
Resolves #56 

Adding a bio for sh-mikem

@chrisbSH

### CODE REVIEW

File: bios/sh-mikem-bio.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#### Name:
+Mike
```
Comment (chrisbSH) [inline bios/sh-mikem-bio.md:2]: Are you sure this is your name?

File: bios/sh-mikem-bio.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+
+#### Location:
+Sunnyvale, CA
```
Comment (ebuceci) [inline bios/sh-mikem-bio.md:5]: Nice location! Just like me!

File: bios/sh-mikem-bio.md (hunk 1 above)
Comment (sh-mikem) [inline bios/sh-mikem-bio.md:2]: Let me double check on that

Comment (mlkebedeUSCIS) [inline bios/sh-mikem-bio.md:2]: trying

**Pull request merged**